proposed/phpdoc.md

```diff
@@ -1220,18 +1220,19 @@ The @method allows a class to know which 'magic' methods are callable.
 
 #### Syntax
 
-    @method [return type] [name]([type] [parameter], [...]) [description]
+    @method <static|> [name]([type] [parameter], [...])[: return type] [description]
```

Comment:
Which brings us back to the issue:
> But honestly, I cannot really figure out the rules how PSR-5 mixes ABNF and PHP :confused:

Obviously the square brackets in `@method [name](…) [description]` have different meaning – or is `name` really optional – or is `description` mandatory?

IMO, an appropriate solution would look something like:
```
@method [static] <name>([<parameter-list>])[: <type>] [<description>]
```

Where square brackets (`[` and `]`) enclose optional elements, and angle brackets (`<` and `>`) denote ABNF non-terminals.